blueocean-core-js/src/js/components/ContentPageHeader.jsx

```diff
@@ -26,7 +27,9 @@ export const SiteHeader = props => {
     return (
         <BasicHeader className="ContentPageHeader">
             <TopNav>
-                <BlueLogo href={props.homeURL} />
+                <Extensions.Renderer extensionPoint="jenkins.header.logo">
+                    <BlueLogo href={props.homeURL} />
```

Comment:
for my culture, again, or can you point to some docs about it: IIUC, the tag in the middle is the one rendered if no extension is found? Like a default value?